%YAML 1.0
---
calibration_time: "Ср 8 сентября 2021 10:22:15"
image_width: 1280
image_height: 720
flags: 0
camera_matrix: !!opencv-matrix
   rows: 3
   cols: 3
   dt: d
   data: [686.3168, 0., 639.5, 0., 686.3168, 359.5, 0., 0., 1.]
distortion_coefficients: !!opencv-matrix
   rows: 1
   cols: 5
   dt: d
   data: [0.0, 0.0, 0.0, 0.0, 0.0]
avg_reprojection_error: 3
